nifi-docs/src/main/asciidoc/expression-language-guide.adoc

```diff
@@ -1385,7 +1385,7 @@ Each of the following functions will encode a string according the rules of the
 *Return Type*: [.returnType]#String#
 
 *Examples*: If we have a URL-Encoded attribute named "url" with the value
-	"https://nifi.apache.org/some%20value%20with%20spaces", then the Expression
+	"https%3A%2F%2Fnifi.apache.org%2Fsome+value+with+spaces", then the Expression
```

Comment:
If the previous example was correct, I think _adding_ this example rather than _replacing_ it makes more sense. Both are valid inputs.

Reply:
Sure, I will make the change. Thanks!